package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "ilib-resbundler",
+  "version": "1.0.0",
+  "description": "convert json resources to js file",
+  "main": "resbundler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
add "dist" target for publishing

Reply:
I added dist target  and also added test files for checking the result by running npm run test-* command